### DISCUSSION TITLE
"Does fooocus have a security hole?"

### DISCUSSION THREAD

**Discussion by Azur231** [lllyasviel/Fooocus#3769] [Q&A] [answered]
Saw this in the Fooocus subreddit and thought it should be addressed.

I’ve been meaning to post this and I should probably start taking precautions.

I’ve been using fooocus for a few months now and absolutely love it. I’ve used pretty much all the other alternatives and always end up going back to fooocus for the simple things.

To explain a little bit, I moved into an apartment with only one option for an ISP. They provide the router and what not. Now this router is a bit different and I honestly hate it. It requires an app to access anything and is pretty limited. However it has built in security “feature” where it will block malicious ads and what not, kinda like that raspberry pi setup does. It also blocks other security events on top of that.

For awhile I ignored it but got surprised when I saw that it blocked someone in China trying to use remote access to get into my main machine. I didn’t think much of it at first but then I noticed a pattern.

It only happens when I run Fooocus. It’s usually pretty quickly into booting it up. It’s now blocked like 10 attempts from all over the world and it’s only ever my main machine and not the other 8 devices.

I never have and never will run it on a public IP/API, but I run it on a local ip so I can use it with my phone sometimes.

Not pointing figures at fooocus directly, but has anyone witnessed anything similar happening? I’m considering removing it and possibly just nuking my SSD just in case It’s mining or eventually going to try and encrypt all my shit.

https://www.reddit.com/r/fooocus/comments/1h7mtow/does_fooocus_have_a_security_hole/

**Comment by mashb1t** [comment, marked as answer]
answered on reddit, TLDR: No.
